.github/workflows/publish.yml

```diff
@@ -48,19 +48,19 @@ jobs:
                 docker system prune -f || :
              fi
           done
-      - name: Build EVE for KVM
+      - name: Build EVE for Xen
         # build #1 without push (do not push either unless both can build)
         run: |
-          rm -rf dist dist.kvm
+          rm -rf dist dist.xen
           make HV=kvm eve
-          mv -f dist dist.kvm
-      - name: Build and push EVE for Xen
+          mv -f dist dist.xen
+      - name: Build and push EVE for KVM
```

Comment:
Also, why are we renaming everything here?

Reply:
Kalyan's logic assumed that snapshot-amd64 was the Xen version, but my previous change reversed the push order, leaving it as KVM. This restores the original order, while still testing that both versions build before pushing either.